subsys/bluetooth/audio/shell/cap_initiator.c

```diff
@@ -501,27 +1166,84 @@ static int cmd_cap_initiator(const struct shell *sh, size_t argc, char **argv)
 	return -ENOEXEC;
 }
 
-SHELL_STATIC_SUBCMD_SET_CREATE(cap_initiator_cmds,
+SHELL_STATIC_SUBCMD_SET_CREATE(
+	cap_initiator_cmds,
 #if defined(CONFIG_BT_BAP_UNICAST_CLIENT)
-	SHELL_CMD_ARG(discover, NULL,
-		      "Discover CAS", cmd_cap_initiator_discover, 1, 0),
+	SHELL_CMD_ARG(discover, NULL, "Discover CAS", cmd_cap_initiator_discover, 1, 0),
 	SHELL_CMD_ARG(unicast-start, NULL,
 		      "Unicast Start [csip] [sinks <cnt> (default 1)] "
 		      "[sources <cnt> (default 1)] "
 		      "[conns (<cnt> | all) (default 1)]",
 		      cmd_cap_initiator_unicast_start, 1, 7),
-	SHELL_CMD_ARG(unicast-list, NULL, "Unicast list streams",
-		      cmd_cap_initiator_unicast_list, 1, 0),
+	SHELL_CMD_ARG(unicast-list, NULL, "Unicast list streams", cmd_cap_initiator_unicast_list,
+		      1, 0),
 	SHELL_CMD_ARG(unicast-update, NULL, "Unicast Update <all | stream [stream [stream...]]>",
-		      cmd_cap_initiator_unicast_update, 2,
-		      CAP_UNICAST_CLIENT_STREAM_COUNT),
+		      cmd_cap_initiator_unicast_update, 2, CAP_UNICAST_CLIENT_STREAM_COUNT),
 	SHELL_CMD_ARG(unicast-stop, NULL, "Unicast stop all streams",
 		      cmd_cap_initiator_unicast_stop, 1, 0),
 	SHELL_CMD_ARG(unicast-cancel, NULL, "Unicast cancel current procedure",
 		      cmd_cap_initiator_unicast_cancel, 1, 0),
+#if UNICAST_SINK_SUPPORTED
+	SHELL_CMD_ARG(ac_1, NULL, "<sink preset>", cmd_cap_ac_1, 2, 0),
+#endif /* UNICAST_SINK_SUPPORTED */
```

Comment:
It's a good place to make use of `SHELL_EXPR_CMD_ARG` macro.
```suggestion
	SHELL_EXPR_CMD_ARG(UNICAST_SINK_SUPPORTED, ac_1, NULL, "<sink preset>", cmd_cap_ac_1, 2, 0),
```
applies in multiple places

Reply:
SHELL_EXPR_CMD_ARG would have been better here, indeed, but suggest to leave as is for now to get it merged